[it] localizzazione termine scalability (#1071)
cncf/glossary: content/it/scalability.md

---
title: Scalabilità
status: Completed

> category: Propriety

Reviewer: ```suggestion
category: Proprietà
```

---

> La scalabilità si riferisce alla capacità di un sistema di estendersi, o altrimenti detto di aumentare la propria capacità di fare qualsiasi cosa il sistema debba fare. Ad esempio, un [cluster](/it/cluster/) [Kubernetes](/it/kubernetes/) si scala aumentando o riducendo il numero delle  applicazioni [containerizzate](/it/containerization/). La scalabilità dipende, però, da diversi fattori. Quanti [nodi](/it/nodes/) ha, quanti [containers](/it/container/) può gestire ogni nodo e quanti record e operazioni può supportare il control plane.

Reviewer: Ho aggiunto le virgolette al termine `record` perché mi pare un concetto connaturato a Kubernetes e difficilmente traducibile (come i log di git, se non ho capito male)
```suggestion
La scalabilità si riferisce alla capacità di un sistema di estendersi, o altrimenti detto di aumentare la propria capacità di fare qualsiasi cosa il sistema debba fare. Ad esempio, un [cluster](/it/cluster/) [Kubernetes](/it/kubernetes/) è scalabile perché vi si possono aumentare o ridurre il numero delle  applicazioni [containerizzate](/it/containerization/). La scalabilità dipende, però, da diversi fattori. Quanti [nodi](/it/nodes/) ha, quanti [containers](/it/container/) può gestire ogni nodo e quanti "record" e operazioni può supportare il pannello di controllo.
```

> Un sistema scalabile facilita l'aggiunta di ulteriore capacità. Si distinguono due approcci di scalabilità. Da un lato, c'è la [scalabilità orizzontale](/it/horizontal_scaling/) che aggiunge altri nodi per gestire un carico maggiore. Al contrario, nella  [scalabilità verticale](/it/vertical_scaling/) i singoli nodi vengono resi più potenti per eseguire più transazioni (ad esempio, aggiungendo più memoria o CPU a una singola macchina). Un sistema scalabile è in grado di cambiare facilmente e di soddisfare le esigenze degli utenti.

Reviewer 2: ```suggestion
Un sistema scalabile facilita l'aggiunta di ulteriore capacità. Si distinguono due approcci di scalabilità. Da un lato, c'è la [scalabilità orizzontale](/it/horizontal_scaling/) che aggiunge altri nodi per gestire un carico maggiore. Dall'altro la [scalabilità verticale](/it/vertical_scaling/) con la quale i singoli nodi vengono resi più potenti per eseguire più transazioni (ad esempio, aggiungendo più memoria o CPU a una singola macchina). Un sistema scalabile è in grado di cambiare facilmente e di soddisfare le esigenze degli utenti.
```
